pkg/jobs/rbac/rbac.go

```diff
@@ -168,6 +253,31 @@ func ApplyRBAC(kubeset kubernetes.Interface, namespace string) error {
 	return nil
 }
 
+func ApplyRBACHypershift(kubeset kubernetes.Interface, namespace string, curatorNamespace string) error {
+	klog.V(2).Info("Check if RoleBinding cluster-installer exists in namespace " + namespace)
+	if _, err := kubeset.RbacV1().RoleBindings(namespace).Get(context.TODO(), "curator", v1.GetOptions{}); err != nil {
+		klog.V(2).Info(" Creating RoleBinding curator in namespace " + namespace)
+		_, err = kubeset.RbacV1().RoleBindings(namespace).Create(
+			context.TODO(), getRoleBinding(curatorNamespace), v1.CreateOptions{})
+		if err != nil {
+			return err
+		}
+		klog.V(0).Info(" Created RoleBinding in cluster namespace ✓")
+	}
+
+	klog.V(2).Info("Check if ClusterRole curator-crb exists")
+	if _, err := kubeset.RbacV1().ClusterRoleBindings().Get(context.TODO(), "curator-crb", v1.GetOptions{}); err != nil {
```

Comment:
I think you only want to do this if the Get returns a not found err. Otherwise return the Get err.